packages/lit/src/index.all.ts

```diff
@@ -36,6 +36,7 @@ export {
   svg as staticSvg,
   unsafeStatic,
   withStatic,
+  StaticValue,
```

Comment:
This is just for a JS bundle, so I don't think adding types here is critical, but probably doesn't hurt.